lint_test/avoid_final_with_getter_test.dart

```diff
@@ -0,0 +1,21 @@
+// ignore_for_file: type_annotate_public_apis, prefer_match_file_name, unused_local_variable
+
+/// Check final private field with getter fail
+/// `avoid_final_with_getter`
+
+class Fail {
+  // expect_lint: avoid_final_with_getter
+  final int _myField = 0;
+
+  int get myField => _myField;
```

Comment:
If the getter name does not match the field name it does not report an issue. I think it would be great to handle this situation as well. What do you think?